disnake/user.py

```diff
@@ -142,8 +152,13 @@ def avatar(self) -> Optional[Asset]:
 
     @property
     def default_avatar(self) -> Asset:
-        """:class:`Asset`: Returns the default avatar for a given user. This is calculated by the user's discriminator."""
-        return Asset._from_default_avatar(self._state, int(self.discriminator) % len(DefaultAvatar))
+        """:class:`Asset`: Returns the default avatar for a given user."""
```

Comment:
It would be nice if the new calculation was documented too with a ``.. versionchanged``

Reply:
Added a few missing `versionchanged`s here, thanks (:
https://github.com/DisnakeDev/disnake/pull/1025/commits/f1000aac3e9277c744caf6da707b2d02b1ca5c50